sdk/test-utils/perfstress/src/utils.ts

```diff
@@ -16,6 +18,27 @@ export function getEnvVar(name: string) {
   return val;
 }
 
+let cachedHttpsAgent: https.Agent;
+/**
+ * Returns https Agent to allow connecting to the proxy tool with "https" protocol.
+ *
+ * @export
+ * @param {string} name
+ */
+export const getCachedHttpsAgent = (insecure: boolean) => {
+  if (!cachedHttpsAgent) {
+    cachedHttpsAgent = new https.Agent({
+      rejectUnauthorized: !insecure
```

Comment:
Why do we need to set both `request.AllowInsecureConnection` above and `rejectUnauthorized` here?

Reply:
allowInsecureConnection is for `core-rest-pipeline` only.
`rejectUnauthorized` is the one we need to tell node's https req.